src/host/main.cpp

```diff
@@ -61,16 +63,8 @@ bool asynchost::TCPImpl::alloc_quota_logged = false;
 size_t asynchost::UDPImpl::remaining_read_quota =
   asynchost::UDPImpl::max_read_quota;
 
-std::chrono::microseconds asynchost::TimeBoundLogger::default_max_time(10'000);
-
-void print_version(size_t)
```

Comment:
Don't think it's safe to remove `print_version` - it's still referenced in the `app.add_flag("-v, --version", print_version, ...)` call?

Reply:
Weird, it seemed unused locally, I'll check.